### PR TITLE
Added Laura's feedback to changelog

### PR DESCRIPTION
**Pull request by zeroZshadow**
#### Description
Updated changelog

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -18,16 +18,16 @@
     - `Send{EVENT_NAME}` is now `Send{EVENT_NAME}Event`.
     - `Send` is now `SendUpdate`.
 - The generated command senders in Monobehaviours have also changed.
```
Comment (paulbalaji) [inline CHANGELOG.md]:
```suggestion
 - The generated command senders in MonoBehaviours have also changed.
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -39,9 +39,9 @@
     - Note that generated types that implement `ISpatialComponentData` are not marked as `Serializable`.
 - Added the `DynamicConverter` class for converting a `ISpatialComponentSnapshot` to an `ISpatialComponentUpdate`.
 - Added a generated ECS shared component called `{COMPONENT_NAME}.ComponentAuthority` for each SpatialOS component.
-    - This component contains a single bool which denotes whether you have authority over that component.
-    - It will not tell you about soft-handover (`AuthorityLossImminent`).
-- You may now `[Require]` `EntityId`, `Entity`, `World`, `ILogDispatcher`, and `WorldCommandSender` in Monobehaviours.
+    - This component contains a single boolean which denotes whether a server-worker instance has write access authority over that component.
+    - The component does not tell you about soft-handover (`AuthorityLossImminent`).
+- You may now `[Require]` an `EntityId`, `Entity`, `World`, `ILogDispatcher`, and `WorldCommandSender` in Monobehaviours.
```
Comment (paulbalaji) [inline CHANGELOG.md]:
```suggestion
- You may now `[Require]` an `EntityId`, `Entity`, `World`, `ILogDispatcher`, and `WorldCommandSender` in MonoBehaviours.
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -75,10 +75,11 @@
 - Reactive components have been isolated and can be disabled.
 - Subscriptions API has been added, this allows you to subscribe anything for which a manager has been defined.
     - This now backs the `Require` API in Monobehaviours
```
Comment (paulbalaji) [inline CHANGELOG.md]:
```suggestion
    - This now backs the `Require` API in MonoBehaviours.
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -17,17 +17,17 @@
...
-    - There is no limit on the number of `GameObject` instances that can be linked to a SpatialOS entity. However the same component type can not be added more than once.
-    - Deleting a linked `GameObject` will unlink it from the SpatialOS entity automatically.
+    - All linked `GameObject` instances are still unlinked before this is called, however it is now your responsibility to track if a `GameObject` was created when the entity was added.
+    - You should now call `linker.LinkGameObjectToSpatialOSEntity()` to link the `GameObject` to the SpatialOS entity.
+    - You should also pass-in a list of `ComponentType` to `LinkGameObjectToSpatialOSEntity` which you wish to be copied from the `GameObject` to the ECS entity associated with the `GameObject`.
+        - Note that for the Transform Synchronization feature module to work correctly, you need to set up a linked Transform GameObject Component on your GameObject. You also need to link a `Rigidbody` GameObject Component if there’s one on your GameObject.
```
Comment (ElleEss) [inline CHANGELOG.md]:
```suggestion
        - Note that for the Transform Synchronization feature module to work correctly, you need to set up a linked Transform Component on your GameObject. You also need to link any Rigidbody Component on your GameObject.
```
I think as we say the name of the Unity component and the fact that we mention GameObject it's clear we mean a GameObject component.
Also we can say Ribidbody outside of backticks as it's a proper word in Unity (https://docs.unity3d.com/Manual/class-Rigidbody.html).

File: CHANGELOG.md [hunk 5]
```diff
@@ -17,17 +17,17 @@
...
+    - There is no limit on the number of GameObject instances that you can link to a SpatialOS entity. However, you cannot add the same component type more than once.
```
Comment (ElleEss) [inline CHANGELOG.md]: Which component type? GameObject? And do we mean link?  If so: 
```suggestion
    - There is no limit on the number of GameObject instances that you can link to a SpatialOS entity. However, you cannot add a component type to a linked GameObject instance more than once.
```

File: CHANGELOG.md [hunk 6]
```diff
@@ -74,11 +74,12 @@
 - Re-added tests for Reader/Writer injection criteria and Monobehaviour enabling.
```
Comment (ElleEss) [inline CHANGELOG.md]:
```suggestion
- Re-added tests for Reader/Writer injection criteria and MonoBehaviour enabling.
```

File: CHANGELOG.md [hunk 7]
```diff
@@ -53,15 +53,15 @@
...
 - Changed the format of the BuildConfiguration asset. Please recreate, or copy it from `workers/unity/Playground/Assets/Config/BuildConfiguration.asset`.
 - Building workers will not change the active build target anymore. The build target will be set back to whatever was set before starting the build process.
 
 ### Fixed
 
-- Fixed a bug where running `SpatialOS -> Generate code` would always regenerate code, even if no files had changed.
-- Fixed a bug where building all workers in our sample projects would fail, if you have Android build support installed, but didn't set the path to the Android SDK.
+- Fixed a bug where, from the SpatialOS menu in the Unity Editor, running **SpatialOS ** > **Generate code** would always regenerate code, even if no files had changed.
+- Fixed a bug where building all workers in our sample projects would fail if you have Android build support installed but didn't set the path to the Android SDK.
 - Fixed a bug where some prefabs would not be processed correctly, causing NullReferenceExceptions in OnEnable.
```
Comment (ElleEss) [inline CHANGELOG.md]:
```suggestion
- Fixed a bug where some prefabs would not be processed correctly, causing `NullReferenceExceptions` in `OnEnable`.
```

**Pull request merged**